reporter/html.js

```diff
@@ -723,6 +723,20 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	// Only add this information if stackTrace is supported
+	if ( Error().stack ) {
```

Comment:
So let's not change `sourceFromStacktrace` and fix this check, either by only looking for `details.source` (I think that's the correct way) or replicating the functionality of `sourceFromStacktrace` correctly. The current check will only output the source location of tests for a subset of browsers (as far as we know).